feature_tests/features/consumer/consumer-countDocumentPointer-success.feature

```diff
@@ -143,67 +143,9 @@ Feature: Consumer Search Success scenarios
       | subject     | 9278693472                     |
       | contentType | application/pdf                |
       | url         | https://example.org/my-doc.pdf |
-    When Consumer "Yorkshire Ambulance Service" counts Document References with query parameters:
-      | property             | value                                               |
-      | subject:identifier   | https://fhir.nhs.uk/Id/nhs-number\|9278693472       |
-      | custodian:identifier | https://fhir.nhs.uk/Id/ods-organization-code\|8FW23 |
-    Then the operation is successful
-    And the response has 1 total
-
-  Scenario: Successfully searches for all documents by type and nhs number for a consumer
```

Comment:
Again, why is this feature test and the one below removed?

Reply:
Type is no longer allowed